CHANGELOG.md

```diff
@@ -8,32 +8,88 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### BREAKING CHANGES
 
-- `@cumulus/api-client/granules.getGranule` now returns the granule record from the GET `/granules/<granuleId>` endpoint, not the raw endpoint response
```

Comment:
I think this note got lost in the merge?

Reply:
Whoops!    Will sort.